processor/transformprocessor/README.md

```diff
@@ -153,6 +154,52 @@ transform:
         - set(body, attributes["http.route"])
 ```
 
+### Context inferred configurations
```

Comment:
Should we update the example above to reflect the new style, even if we leave in explicitly setting `context`?

Reply:
Good question, maybe we should not changing it yet so it includes more examples of non-inferred configurations. I'll undo the changes. Thanks!

Reply:
Oh no, I'm thinking we should just update everything to include contexts in paths. We should try to nudge people toward the new style as much as possible.

Reply:
NP, reverting the revert... :)